### DISCUSSION TITLE
Metamask connection is no poping up in mobile metamask app

### DISCUSSION THREAD

**Discussion by kottackalsulvin** [WalletConnect/walletconnect-monorepo#807] [Developer Support]
Hi,
I am trying to connect dapp with mobile metamask app using wallet connect but the metamask connection pop up is not showing in mobile app. Anyone please help me to resolve this issue.

**Here is my web3 code:**

```
this.provider = new WalletConnectProvider({
infuraId: "2a0445aca6a8423aae67ff63f9718f6a",
qrcodeModalOptions: {
  mobileLinks: [
    "metamask",
  ],
},
});
```

**Environment:**
@walletconnect/web3-provider: 1.4.1
mobile metamask version: v4.1.0

**Comment by andiskim** [comment]
same here, the connection only happens sometimes, can't figure out why

**Comment by kottackalsulvin** [comment]
It was working 4 months ago but now it is not at all working. Anyone pls help me to figure out this?
